docker-compose.override.yml

```diff
@@ -35,6 +35,7 @@ services:
       - ASPNETCORE_URLS=http://+:80
       - NetPodcastApi__BaseAddress=http://podcast.api
       - ConnectionStrings__ListenTogetherDb=Server=podcast.db;Database=ListenTogether;User Id=sa;Password=Pass@word
```

Comment:
I don't think we should have passwords, or connection strings in source control.

Reply:
if it's only for the local scenario i hate it less than if it's "guidance," but i do have a modicum of concern that we'd send the message that hard-coding strings is a good idea, and we know it isn't. Could we compromise and maybe add a .env file with guidance on how to fill it in?

Reply:
yeah I like that approach. someone want to do that ;)